sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12-integration.module.ts

```diff
@@ -4,20 +4,20 @@ import { ConfigModule } from "@sims/utilities/config";
 import { IER12IntegrationService } from "./ier12-integration.service";
 import { IER12FileService } from "./ier12-file.service";
 import {
+  IntegrationStudentAssessmentService,
   SshService,
-  StudentAssessmentService,
 } from "@sims/integrations/services";
 
 @Global()
 @Module({
   imports: [ConfigModule],
   providers: [
     SshService,
-    StudentAssessmentService,
+    IntegrationStudentAssessmentService,
```

Comment:
Why did we rename the StudentAssessmentService prefixing Integration? 

If we follow this in one place, then we must use the same pattern everywhere. 

@andrepestana-aot  @andrewsignori-aot @guru-aot ?

Reply:
Resolving as a team agreement was made.